src/games/chess/sound.ts

```diff
@@ -0,0 +1,6 @@
+import MoveSound from './media/move.mp3';
+
+export const playSound = () => {
+  const sound = new Audio(MoveSound);
```

Comment:
Does this mean it loads the file every time it tries to play? If I am not mistaken we used to store the Audio as a global in other games?